src/tests/ftest/util/server_utils_params.py

```diff
@@ -329,7 +329,8 @@ class PerEngineYamlParameters(YamlParameters):
         REQUIRED_ENV_VARS = {
             "common": [
                 "D_LOG_FILE_APPEND_PID=1",
-                "COVFILE=/tmp/test.cov"],
+                "COVFILE=/tmp/test.cov",
+                "DAOS_SCHED_WATCHDOG_ALL=1"],
```

Comment:
Please see my comment on https://daosio.atlassian.net/browse/DAOS-10131
I would prefer if this can be overridden by individual test cases.
Phil suggests it could be added here b default, which would allow overriding: https://github.com/daos-stack/daos/blob/c4a0f87fac0f58fa0c233d4580aab8ad75073d18/src/tests/ftest/util/server_utils_params.py#L389-L394